lib/executor/helpers.go

```diff
@@ -33,6 +33,32 @@ func getStagesUnscaledMaxTarget(unscaledStartValue int64, stages []Stage) int64
 	return max
 }
 
+func validateNumberOfVuShifts(startVus int64, stages []Stage) []error {
+	const MaxRampingVUShift = 100000000
+
+	var errors []error
+	totalShifts := big.NewInt(startVus)
+	prevNumVus := totalShifts
+
+	for _, s := range stages {
+		current := big.NewInt(s.Target.Int64)
+		vuDelta := new(big.Int).Sub(current, prevNumVus)
+		vuDelta.Abs(vuDelta)
+
+		totalShifts.Add(totalShifts, vuDelta)
+
+		prevNumVus = current
+	}
+
+	totalShifts.Add(totalShifts, prevNumVus)
```

Comment:
I'm not sure I grasp why we're adding to `totalShifts` again here. Could you clarify, and if relevant, could you add a comment detailing why we do it?